doc/getting-started/install.md

```diff
@@ -75,21 +75,22 @@ Confirm that you installed cabal version `3.4.0.0`:
 
 #### Downloading and installing the GHC compiler:
 
-Create a working directory for your builds:
-
-    mkdir -p ~/src
-    cd ~/src
+Download and install version `8.10.4` of GHC.  The easiest way to do this is to use [ghcup](https://www.haskell.org/ghcup/).
 
- Download and install the latest version of GHC:
+```bash
+$ curl --proto '=https' --tlsv1.2 -sSf https://get-ghcup.haskell.org | sh
+$ ghcup install ghc 8.10.4
```

Comment:
You might need to source the bashrc to set the env variables

Reply:
Good point.  I've updated the instructions.